common/searchattribute/search_attirbute.go

```diff
@@ -48,6 +48,11 @@ type (
 	Saver interface {
 		SaveSearchAttributes(indexName string, newCustomSearchAttributes map[string]enumspb.IndexedValueType) error
 	}
+
+	Manager interface {
+		Provider
+		Saver
```

Comment:
Getter & Setter?

saver sounds weird

Reply:
usually getter <-> setter, provider <-> consumer.

not sure what is the preferable term here

Reply:
I removed `Saver` interface. It is not actually needed.